src/data/endpoints/managed.yaml

```diff
@@ -238,6 +238,36 @@ endpoints:
             curl -H "Authorization: Bearer $TOKEN" \
                 -X DELETE
                 https://$api_root/$version/managed/services/$service_id
+  /managed/services/$id/enable:
+    group: Services
+    type: Action
+    authenticated: true
+    description: >
+      Enables a Managed service your account may access.
+    methods:
+      POST:
+        description: >
+          Enables this Managed service.
```

Comment:
You should add what this actually means - enabled services are actively monitored, and maybe that this only really does anything if the service in question is presently disabled.